OA-PASS/pass-ui
# Refactoring static assets / pages

Related: https://github.com/eclipse-pass/main/issues/169

* Copy various static assets to Ember app
* Disable static-html in dev docker setup 
* Don't render any references to static pages, such as contact or FAQ pages, that appear in various places in the app
* Update tests to reflect updated config.json

The updated environment variables would have to be ported over to `pass-docker` after this PR is merged, but before a new `pass-ui` image can be generated. 

### Remaining issues
* There are still some references specifically to Johns Hopkins University, especially the institutional OA policy. Not in scope for this ticket to change this
* Need to update the top level proxy to not redirect traffic to `static-html` once that is removed from `pass-docker`. That'll likely be done after this PR is merged, once a new `pass-ui` image is ready to be created. That new proxy image would then have to be brought back into the `pass-ui` local dev environment.

### Manual Testing
This assumes you have an entry in your system's `hosts` file that points `pass.local` to `localhost`
* Boot up the docker env in `.docker/`. 
* The application should be available at `pass.local/app`
* The site should look the same, without nav links to contact, about, or faq pages.
* Make sure there is no notification on screen about failing to load the theme
